mobile-widgets/qmlmanager.cpp

```diff
@@ -339,10 +339,6 @@ void QMLManager::openLocalThenRemote(QString url)
 		appendTextToLog(QStringLiteral("%1 dives loaded from cache").arg(dive_table.nr));
 		setNotificationText(tr("%1 dives loaded from local dive data file").arg(dive_table.nr));
 	}
-	if (qPrefCloudStorage::cloud_verification_status() == qPrefCloudStorage::CS_NEED_TO_VERIFY) {
-		appendTextToLog(QStringLiteral("have cloud credentials, but still needs PIN"));
```

Comment:
so we are missing this fairly useful piece of information in the log

Reply:
it's back.